spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRequiredDistributionAndOrdering.java

```diff
@@ -298,4 +298,21 @@ public void testRangeDistributionWithQuotedColumnNames() throws NoSuchTableExcep
         ImmutableList.of(row(7L)),
         sql("SELECT count(*) FROM %s", tableName));
   }
+
+  @Test
+  public void testDefaultSortOnBinaryColumn() {
```

Comment:
Is there another place to add a unit test? This test file does not seem directly related.

Reply:
Updated to the dedicated test class.